src/main.ts

```diff
@@ -703,13 +703,15 @@ export default class DiceRollerPlugin extends Plugin {
                 return roller;
             }
             case "table": {
-                return new TableRoller(
+                const roller = new TableRoller(
                     this,
                     content,
                     lexemes[0],
                     source,
                     showDice
                 );
+                await roller.initPromise;
```

Comment:
Is there a reason for making `this.initPromise` vs. just directly calling `this.getFile()`?